crates/turbopack-test-utils/Cargo.toml

```diff
@@ -9,6 +9,9 @@ autobenches = false
 [lib]
 bench = false
 
+[lints]
+workspace = true
```

Comment:
Do we have to remember to add this to every crate?

Reply:
Yeah, it seems so. Unfortunately I couldn't find a way around this.

https://blog.rust-lang.org/2023/11/16/Rust-1.74.0.html#lint-configuration-through-cargo

Reply:
One neat thing is `cargo new` seems to add it to new crates it generates in the workspace.